generation/WinSDK/enums.json

```diff
@@ -28818,6 +28804,132 @@
         "interface": "IDxcUtils",
         "method": "LoadFile",
         "parameter": "pCodePage"
+      },
+      {
+        "struct": "DxcBuffer",
+        "field": "Encoding"
+      }
+    ]
+  },
+  {
+    "name": "DxcValidatorFlags",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "DxcValidatorFlags_",
+      "header": "dxcapi.h"
+    },
+    "uses": [
+      {
+        "interface": "IDxcValidator",
+        "method": "Validate",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "IDxcValidator2",
+        "method": "ValidateWithDebug",
+        "parameter": "Flags"
+      }
+    ]
+  },
+  {
+    "name": "DxcVersionInfoFlags",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "DxcVersionInfoFlags_",
```

Comment:
Unfortunately both these `static const UINT32`s don't appear to be parsed by `enums.json`?

Reply:
Seems more like a ClangSharp limitation than enums.json.

Reply:
Yeah, either way it's something that I expected to show up, but it didn't. Should I manually declare `"members"`, while leaving `"autoPopulate"` in place in hopes of getting a compilation conflict when ClangSharp can provide these to us, or is it a fix you can do elsewhere in the tooling?

As this PR shows we used to have manual fields for `DXC_CP_` too (now unnecessary), while also relying on `autoPopulate` without conflicts being generated?

Reply:
Yes you can manually declare the missing constants together with autoPopulate. The sets will be merged.

Reply:
Cool. Fixed it by removing it, because they can be autodetected.

I don't remember the status of this PR: I think I got stuck on the `static const UINT32`s not being parsed, only `#define`s? Is there any plan/solution for that?

Reply:
> Seems more like a ClangSharp limitation than enums.json.

Am I right to assume that `ClangSharp` emits whatever it finds _after preprocessing_? I'd assume that all the preprocessor directives are flattened out, which is why the `ConstantsScraper` specifically looks for `#define` in source files:

https://github.com/microsoft/win32metadata/blob/2e6b61977d4b7f5f852637bb7c2d121d1d687a96/sources/MetadataUtils/ConstantsScraper.cs#L32-L34

This is the only code that looks at `autoPopulate`. Assuming that "actual C constants" are extracted from ClangSharp and written elsewhere (could it be [`ConstantWriter`](https://github.com/microsoft/win32metadata/blob/main/sources/MetadataUtils/ConstantWriter.cs)?), is this where we should insert extra logic to turn `static const` definitions into grouped `enum`s as well?